src/graceful_termination.jl

```diff
@@ -169,3 +200,19 @@ function graceful_terminate(pid::Int32=entrypoint_pid(); wait::Bool=true)
 
     return nothing
 end
+
+function install_supervise_shim(shims_root::AbstractString; use_symlink=false)
```

Comment:
We should probably prefer using symlinks as if the package is uninstalled the symlink will be broken
```suggestion
function install_supervise_shim(shims_root::AbstractString; use_symlink=true)
```